cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -4,10 +4,7 @@
 from cfme.utils.log import logger
 from cfme.utils.wait import wait_for
 
-pytestmark = [
-    pytest.mark.ignore_stream("upstream"),
-    test_requirements.ansible
-]
+pytestmark = [pytest.mark.ignore_stream("upstream"), test_requirements.ansible]
```

Comment:
please don't reformat code w/o special need

Reply:
It happened due to Black changes. Changing it.